rogi_rl/cli.py

```diff
@@ -22,5 +22,21 @@ def demo(width, height):
     server.launch()
 
 
+@click.command()
+@click.option('-p/-np', '--profile-render/--no-profile-render',
```

Comment:
Should we just use something like : `--render-on` ?
It will be more intuitive to understand.

Reply:
I have made the changes